src/main/java/de/mediathekview/mserver/crawler/srf/SrfConstants.java

```diff
@@ -0,0 +1,33 @@
+package de.mediathekview.mserver.crawler.srf;
+
+public final class SrfConstants {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)